test-tools/src/main/java/io/neow3j/test/NeoTestContainer.java

```diff
@@ -72,12 +72,69 @@ public NeoTestContainer(String configFileSource) {
         }
     }
 
-    public static String getResultFilePath(String testClassName, String methodName) {
-        return "responses/" + testClassName + "/" + methodName + ".json";
+    @Override
+    public TestBlockchain withSecondsPerBlock(int secondsPerBlock) {
+        return null;
     }
 
+    @Override
+    public TestBlockchain withBatchFile(String batchFile) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public TestBlockchain withCheckpoint(String checkpointFile) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public TestBlockchain withConfigFile(String configFile) {
+        return null;
```

Comment:
So, all the methods will `null` here are, in fact, not implemented, right?

If so, can you throw `java.lang.UnsupportedOperationException`?

Reply:
or, maybe it's supported, but it's not implemented by us yet?

In this case, I would throw another exception.

Reply:
Ouh, it was not my intention to push the changes on `NeoTestContainer` 😟 
We should have a call about this, deciding if it even makes sense to have `NeoTestContainer` implement `TestBlockchain`.